docs/versioning.md

```diff
@@ -215,6 +217,39 @@ the following rules:
   provided along with each Antrea release, until the API version is removed
   altogether.
 
+## Introducing new API resources
+
+### Introducing new CRDs
+
+Starting with Antrea v1.0, all Custom Resource Definitions (CRDs) for Antrea are
+defined in the same API group, `crd.antrea.io`, and all CRDs in this group are
+versioned individually. For example, at the time of writing this (v1.3 release
+timeframe), the Antrea CRDs include:
+
+* `ClusterGroup` in `crd.antrea.io/v1alpha2`
+* `ClusterGroup` in `crd.antrea.io/v1alpha3`
+* `Egress` in `crd.antrea.io/v1alpha2`
+* etc.
+
+Notice how 2 versions of `ClusterGroup` are supported: the one in
+`crd.antrea.io/v1alpha2` was introduced in v1.0, and is being deprecated as it
+was replaced by the one in `crd.antrea.io/v1alpha3`, introduced in v1.1.
+
+When introducing a new version of a CRD, [the API deprecation policy should be
+followed](#apis-deprecation-policy).
+
+When introducing a CRD, the following rule should be followed in order to avoid
```

Comment:
maybe also state that for all other non overlapping CRDs to be introduced, we start with v1alpha1?